homeworks/asztaloslajos/3_conditionals_loop/flat_finder.py

```diff
@@ -0,0 +1,13 @@
+"""
+flat_finder.py --- Asztalos Lajos --- 2024.10.12
+"""
+#Input
+city = input("Város:").strip().title()
+apartment_rent = int(input("Bérleti díj dollárban:"))
+#Conditional
+if city == "Washington":
+    print(f"Washingtonba {apartment_rent} dollár bérleti díjjért sem költözhetsz, mert utálod ezt a várost")
+elif ((city == "New York" or city == "San Fransisco") and apartment_rent < 4000) or apartment_rent <=3000 or city == "Chicago":
```

Comment:
Nem hiba, csak egy rövidítési lehetőség:

`city in ['San Francisco', 'New York']`, ezzel kicsit egyszerűsíteni tudod az elif-et